server.js

```diff
@@ -19,11 +13,124 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+app.get("/", (request, response) => {
+response.status(200).json({
+    Hello: "Here you can see all my routes!",
+    Routes: [
+      { "/movies": "all movie data" },
+      { "/titles": "a list of all titles" },
+      { "/movies/:id": "E.g. if you add 81193313 as the id, it will show information about a specific movie, for instance id 81193313 will give us info about the movie Chocolate" },
+      { "/movies/year/:year": "a list of movie data from a specific release year" },
+    ],
+  });
+});
+//get all data
+app.get("/movies", (request, response) => {
+  console.log('Movies...')
+  if (netflixData) {
+    response.status(200).json({ 
+      data: netflixData, 
+      success: true,
+      message: "OK",
+      debug: 'Movies' });
+  } else {
+    response.status(200).json({ 
+      data: [], 
+      success: false,
+      message: "Not Found"});
+  }
+});
+
+//get all titles
+
+app.get("/titles", (request, respons) => {
+  const titleList = netflixData.map((movie) => movie.title);
```

Comment:
this was super neat with the titles, and displaying them all in a list. Will be stealing this ;-)